hif/pcie/8964/tx_ndp.c

```diff
@@ -336,9 +336,8 @@ int pcie_tx_init_ndp(struct ieee80211_hw *hw)
 
 	if (sizeof(struct pcie_tx_ctrl_ndp) >
 	    sizeof(tx_info->driver_data)) {
-		wiphy_err(hw->wiphy, "driver data is not enough: %d (%d)\n",
-			  sizeof(struct pcie_tx_ctrl_ndp),
-			  sizeof(tx_info->driver_data));
+		wiphy_err(hw->wiphy, "driver data is not enough: %zu (%zu)\n",
+			  sizeof(struct pcie_tx_ctrl_ndp), sizeof(tx_info->driver_data));
```

Comment:
sizeof checks can be done at compile time `BUILD_BUG_ON`

Reply:
Not sure I follow